code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1305,6 +1305,22 @@ GLOBAL_LIST_EMPTY(custom_outfits) //Admin created outfits
 			var/turf/startT = spaceDebrisStartLoc(startside, T.z)
 			var/turf/endT = spaceDebrisFinishLoc(startside, T.z)
 			new /obj/effect/immovablerod(startT, endT,target)
+		if(ADMIN_PUNISHMENT_SUPPLYPOD)
+			///////load the supply pod up with something!
+			var/target_path = input(usr,"Enter typepath of an atom you'd like to send with the pod (type \"cancel\" to abort smiting):" ,"Typepath","/obj/item/reagent_containers/food/snacks/grown/harebell")
+			if (target_path == "cancel")//if you hit cancel, then stop
+				return
+			if (target_path == "")//if you dont type anything, spawn an empty pod
+				new /obj/effect/DPtarget(get_turf(target), null, 1)

```

Comment:
Is this intended to be 2 (`POD_CENTCOM`)? Should probably use the define here and on line 1323.